.eslintrc

```diff
@@ -223,7 +223,6 @@
     //
     // React specific linting rules for ESLint
     //
-    "react/display-name": 1, // Prevent missing displayName in a React component definition
```

Comment:
It is unnecessary when using JSX: https://facebook.github.io/react/docs/component-specs.html#displayname
Please correct me if I am wrong.